src/python/turicreate/test/test_sarray.py

```diff
@@ -11,6 +11,7 @@
 from ..data_structures.sarray import SArray
 from ..data_structures.sframe import SFrame
 from ..data_structures.sarray import load_sarray
+from ..toolkits.image_analysis import image_analysis
```

Comment:
let's don't introduce toolkit code in this stage since sarray is the foundation of other high-level application code. Let's make it just self-contained by just using bare flexible type and sarray itself. Use the image type defined in data_structure.Image to explicit add image to sarray.

Reply:
Ok, I just check the implementation. The actual image load happens inside our cpp backend.  You are good to go. Better use `from ... import load_images` to only bind name load_image to the current namespace. So far so good! Great work!